plugins/vli/fu-vli-usbhub-device.c

```diff
@@ -771,24 +776,39 @@ fu_vli_usbhub_device_ready(FuDevice *device, GError **error)
 	}
 
 	/* detect update protocol from the device ID */
-	switch (fu_struct_vli_usbhub_hdr_get_dev_id(self->st_hd1) >> 8) {
+	switch (fu_struct_vli_usbhub_hdr_get_dev_id(self->st_hd1)) {
 	/* VL810~VL813 */
-	case 0x0d:
+	case 0x0d12:
```

Comment:
s that enough? e.g. is there ever a 0xd11 that would have matched before but not now?

Reply:
in our HW design, for 0x0d case, we only have 0x0d12.